.github/workflows/pr-monitor.yml

```
@@ -0,0 +1,143 @@
+        id: pr_status
+        run: |
+          # Get all open PRs
+          PRS=$(gh pr list --json number,title,headRefName,reviewDecision,mergeable,mergeStateStatus --jq '.')
+          echo "prs=$PRS" >> $GITHUB_OUTPUT
+          
+          # Check dependencies between PRs
+          for PR in $(echo "$PRS" | jq -r '.[] | @base64'); do
+            PR_DATA=$(echo "$PR" | base64 --decode)
+            PR_NUM=$(echo "$PR_DATA" | jq -r '.number')
+            PR_BODY=$(gh pr view "$PR_NUM" --json body -q '.body')
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)